Neos.Fusion/Resources/Private/Fusion/Deprecated/Attributes.fusion

```diff
@@ -0,0 +1,15 @@
+# Renders attributes of a HTML tag
+#
+# Usage:
+# attributes = Neos.Fusion:Attributes {
+#   foo = 'bar'
+#   class = Neos.Fusion:DataStructure {
+#     class1 = 'class1'
+#     class2 = 'class2'
+#   }
+# }
+#
+# @deprecated since Neos 5.0 in favor of Neos.Fusion:Tag or Neos.Fusion:Join
```

Comment:
Shouldn't this be `in favor of Neos.Fusion:DataStructure`? Optionally with `@sortProperties = false`?

Reply:
i guess - actually there is no real 1to1 replacement

i would suggest:
`in favor of Neos.Fusion:Tag internal attributes handling when a datastructure is passed`

Reply:
I took the deprecation notice from the PHP code. But I also prefer your version.